app/assets/stylesheets/application.css.scss

```diff
@@ -13,69 +14,121 @@ body {
   width: 100vw;
   height: 100vh;
   font-family: 'Roboto', sans-serif;
-  // padding-top: 65px;
-  background: linear-gradient(180deg, #4d9450, #5a97b4, #5a60b7);
-  background-size: 600% 600%;
+  color: #e8ede8 !important;
+}
 
-  -webkit-animation: AnimationName 12s ease infinite;
-  -moz-animation: AnimationName 12s ease infinite;
-  animation: AnimationName 12s ease infinite;
+.background {
```

Comment:
really excited to see all of this in production, you have done so much! thanks for putting in the time to get it mobile ready and functional!!!